python/kserve/kserve/model.py

```diff
@@ -78,7 +78,7 @@ def __init__(self, name: str):
         self._grpc_client_stub = None
         self.enable_latency_logging = False
 
-    async def __call__(self, body: Union[Dict, CloudEvent, ModelInferRequest],
+    async def __call__(self, body: Union[bytes, Dict, CloudEvent, ModelInferRequest],
```

Comment:
Also need to update the type hint for the `preprocess` handler.

Reply:
Fixed